examples/domainmodel/example/blog.dmodel

```diff
@@ -1,7 +1,6 @@
-datatype String
-
 entity Blog {
     title: String
+    date: complex.Date
```

Comment:
This change broke the CLI: running `./bin/cli generate example/blog.dmodel` in the terminal no longer works because the datatype references can't be resolved.

How shall we handle this? Automatically include other `.dmodel` files in the same directory? Add a CLI option to specify a root path?

Reply:
> > This change broke the CLI: running ./bin/cli generate example/blog.dmodel
>
> Is it possible to setup a CLI specific test case for that? 

Yes, we could add a test that runs the CLI, checks the exit code, and then checks whether the expected output file exists.

>> Add a CLI option to specify a root path?
>
> We should run the indexer in the source root folder/-s which is/are either specified by the user or using a good default

:+1: